src/pages/PreviewScreen.jsx

```diff
@@ -52,30 +72,57 @@ const PreviewScreen = () => {
 
   const isPreview = location.pathname.startsWith("/preview");
```

Comment:
is this additional check needed? if urlRoomId='preview' it is preview i think

Reply:
yes, because if the URl is `https://<subdomain>/<room-id>`
then it'll redirect to `https://<subdomain>/preview/<room-id>`
check out line 82